x-pack/elastic-agent/magefile.go

```diff
@@ -625,6 +630,10 @@ func copyAll(from, to string) error {
 	})
 }
 
+func dockerBuild(tag) string {
```

Comment:
Type needs to be added
```suggestion
func dockerBuild(tag string) string {
```